libbeat/processors/actions/decode_json_fields_test.go

```diff
@@ -580,9 +580,9 @@ func TestAddErrorToEventOnUnmarshalError(t *testing.T) {
 }
 
 func getActualValue(t *testing.T, config *conf.C, input mapstr.M) mapstr.M {
-	log := logp.NewLogger("decode_json_fields_test")
+	log := logptest.NewTestingLogger(t, "decode_json_fields_test")
 
-	p, err := NewDecodeJSONFields(config)
+	p, err := NewDecodeJSONFields(config, logptest.NewTestingLogger(t, ""))
```

Comment:
[Blocker]
Why don't you reuse the same logger? Or create a child logger? I don't think it makes sense to create 2 distinct loggers for the same function.